src/main/webapp/app/shared/models/frontend/sidenav/form-sidenav-configuration.ts

```diff
@@ -47,4 +49,14 @@ export interface FormSidenavConfiguration {
    * Used to hide the save action
    */
   hideSaveAction?: boolean;
+
+  /**
+   * Used to hide the slide toggle button
+   */
+  displaySlideToggleButton?: boolean;
+
+  /**
+   * Function called when the slide toggle button is pressed
+   */
+  slideToggleButtonPressed?: (event: MatSlideToggleChange, formGroup: FormGroup, formField: FormField[]) => void;
```

Comment:
I think it would be better to wrap this in a SlideToggleButtonConfiguration file

Reply:
Fixed: The toggle slide button configuration has been moved into a dedicated configuration file